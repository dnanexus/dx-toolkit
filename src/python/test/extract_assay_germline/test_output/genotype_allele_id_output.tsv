sample_id	allele_id	locus_id	chr	pos	ref	alt	genotype_type
sample_1_3	18_47361_T_G	18_47361_T_C	18	47361	T	G	het-alt
sample_1_4	18_47361_T_G	18_47361_T_C	18	47361	T	G	het-alt
