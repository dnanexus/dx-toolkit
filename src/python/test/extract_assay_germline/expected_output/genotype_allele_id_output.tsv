	sample_id	a_id	locus_id	chr	pos	ref	alt	type
8	sample_1_3	18_47361_T_G	18_47361_T_C	18	47361	T	G	het-alt
9	sample_1_4	18_47361_T_G	18_47361_T_C	18	47361	T	G	het-alt
